app/analysis/index/template.hbs

```diff
@@ -309,4 +316,4 @@
       </div>
     {{/with}}
   {{/if}}
-{{/if}}
\ No newline at end of file
+{{/if}}
```

Comment:
Where is the extra `if` coming from?

Reply:
Right - I'm just confused about the diff: It looks like you added a closing `if` without adding an opening one?

Reply:
Vim added an EOL, which the previous editor didnt.

Reply:
Oh right. We should add https://eslint.org/docs/rules/eol-last and run `--fix` to enforce consistent style.

Reply:
Can you do that in a separate MR?

Reply:
So add EOLs or remove them?

Reply:
It seems that the Unix default is to add them (just like the Vim and the ESlint default), so i'd say let's add them :wink: